utils/kata-manager.sh

```diff
@@ -269,16 +269,20 @@ containerd_installed()
 
 pre_checks()
 {
+	local only_kata="${1:-}"
+
 	info "Running pre-checks"
 
 	command -v "${kata_shim_v2}" &>/dev/null \
 		&& die "Please remove existing $kata_project installation"
 
-	local ret
+	if [ "$only_kata" != "true" ]; then
+		local ret
 
-	{ containerd_installed; ret=$?; } || true
+		{ containerd_installed; ret=$?; } || true
```

Comment:
Check the use of space here instead of tab.

Reply:
Here is tab actually 🧐 I'm not sure why it was highlighted